Home/Home 3.2/Triangle.swift

```diff
@@ -0,0 +1,32 @@
+import Foundation
+
+class Triangle {
+    let a: Double
+    let b: Double
+    let c: Double
+
+    init(a: Double, b: Double, c: Double) {
+        self.a = a
+        self.b = b
+        self.c = c
+    }
+
+    func area() -> Double {
+        let s = (a + b + c) / 2
+        return sqrt(s * (s - a) * (s - b) * (s - c))
+    }
+    
+    func height(for side: Character) -> Double? {
+        let area = self.area()
+        switch side {
+        case "a":
```

Comment:
I hope it should be enum to strictly limit the possible inputs here

Reply:
Refactored the code in [90ac7f0](https://github.com/YaroslavKSE/Professinal-Programming/pull/10/commits/90ac7f03ae60329ab88db27d18a6b1b2a16da546)